docs/en/03_How_tos/adding_a_calendar.md

```diff
@@ -69,32 +75,36 @@ later) uses internally.
 Now that we have the parameters available, we can combine that with `AllEvents` to fetch the data and show it in the
 template. Let's create a getter in the `Page_Controller` (or wherever you are attempting to create the calendar).
 
-	:::php
-	public function SiteEvents() {
-		$params = $this->parseEventParams();
-		return EventHolder::AllEvents(null, null, null, null, $params['year'], $params['month']);
-	}
+```php
+public function SiteEvents() 
+{
+    $params = $this->parseEventParams();
+    return EventHolder::AllEvents(null, null, null, null, $params['year'], $params['month']);
+}
+```
 
 We can now access the events for the currently selected (or default) month in the template, say `Page.ss`. Here is how
 to iteratively loop through the events:
 
-	:::ss
-		<h3>Site Events</h3>
-		<% loop SiteEvents %>
-			<div class="event-listing">
-				<h4><a href="$Link">$Title</a></h4>
-				<p>$Date.Nice $StartTime.Nice - $EndTime.Nice</p>
-			</div>
-		<% end_loop %>
+```ss
```

Comment:
Does this work? I'm thinking `html` is a sensible fallback if it doesn't

Reply:
It hopefully will one day, we need to PR the language pack into https://github.com/github/linguist. It falls back to none which is fine IMO.